ibis/tests/sql/test_compiler.py

```diff
@@ -2298,6 +2298,41 @@ def test_table_difference(self):
 SELECT `string_col` AS `key`, `double_col` AS `value`
 FROM functional_alltypes
 WHERE `int_col` <= 0"""
+
+        assert result == expected
+
+    def test_intersect_project_column(self):
+        # select a column, get a subquery
+        intersection = self._case_intersect()
+        expr = intersection[[intersection.key]]
+        result = Compiler.to_sql(expr)
+        expected = """SELECT `key`
+FROM (
+  SELECT `string_col` AS `key`, CAST(`float_col` AS double) AS `value`
+  FROM functional_alltypes
+  WHERE `int_col` > 0
+  INTERSECT
+  SELECT `string_col` AS `key`, `double_col` AS `value`
+  FROM functional_alltypes
+  WHERE `int_col` <= 0
+) t0"""
```

Comment:
What do you think about creating a constant with this, with the `INTERSECT` / `EXCEPT` as variable, and in the tests just use `QUERY.format('INTERSECT')`? Besides avoiding the duplication, I think it should make tests more readable.

Reply:
done. Good idea much nicer now